.github/workflows/release.yml

```diff
@@ -16,20 +16,54 @@ jobs:
           ref: main
           fetch-depth: 0
 
+
+      - name: Install xmlstarlet
+        run: sudo apt-get update && sudo apt-get install -y xmlstarlet
```

Comment:
Here you install it for release workflow, but it's not needed to be installed in `openapi-update-generate-new-sdk.yml` since you are using it also at this place?

Reply:
It is needed in both workflows to edit files by updating the version.